DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/classes/DSACampaign.java

```diff
@@ -14,38 +14,69 @@
 
 package com.google.sps.classes;
 
-// contains all the necessary information about a DSA campaign, including id's, input settings, and statistics
+// contains all the necessary information about a DSA campaign
 public class DSACampaign {
 
-	public int DSACampaignId;
-    public int userId;
-    public int keywordCampaignId;
+    // unique identifier for the DSA campaign
+	public String DSACampaignId;
+    // id of the user who created the DSA campaign
+    public String userId;
+    // id of the keyword campaign this DSA campaign was run on
+    public String keywordCampaignId;
 
     // settings
+
+    // name of the DSA campaign (used for display purposes)
 	public String name;
-    public String fromDate;
-    public String toDate;
+    // status of the blackbox data retrieval: pending (stats still pending) or complete (stats retrieved)
+    public String campaignStatus;
```

Comment:
Why not use an enum?

Reply:
I don't think datastore supports enums.

Reply:
Ok, I would also suggest adding a TODO to validate that the inputs are what you expect.

Reply:
Added